### PR TITLE
Fix Nvidia driver ignoring wglSwapIntervalEXT

### PR DESCRIPTION
**Pull request by dpjudas**

### CODE REVIEW

File: src/win32/win32gliface.cpp [hunk 1]
```diff
@@ -1141,7 +1141,7 @@ void Win32GLFrameBuffer::ReleaseResources ()
 
 void Win32GLFrameBuffer::SetVSync (bool vsync)
 {
-	if (vsyncfunc != NULL) vsyncfunc(vsync);
+	if (vsyncfunc != NULL) vsyncfunc(vsync ? 1 : 0);
```
Comment (edward-san) [inline src/win32/win32gliface.cpp]: is this change really needed?

Comment (dpjudas) [inline src/win32/win32gliface.cpp]: Strictly speaking no, assuming that all C++ compilers actually map bool to 0 and 1. In any case, I personally consider it a bad practice to pretend something is a bool when it isn't - wglSwapInterval takes an 'swap interval' integer, not a boolean.

Comment (Mateos81) [inline src/win32/win32gliface.cpp]: In C, C++ and Java, the boolean type IS an integer, if you look low-level.

Comment (dpjudas) [inline src/win32/win32gliface.cpp]: I know, but the argument to wglSwapInterval is not a boolean in the semantic meaning. For example, 2 is a valid value to this function. By passing the bool directly it gives the reader of the code the idea that it takes a true/false argument when it in fact does not.

I originally made the change because I wasn't 100% sure whether a C++ true value always maps to 1 (seems it does), but nevertheless I kept the change because I find it semantically wrong.
